### PR TITLE
[ui][new] kudu connector test 16x

### PR DESCRIPTION
**Pull request by mcada**
//test: `@integrations-kudu`

### CODE REVIEW

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by mcada** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Comment by avano** [comment]
retest this please

**Comment by syndesisqe** [comment]
Build finished. Tests FAILED

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

File: utilities/src/main/resources/templates/syndesis-kudu.yml [hunk 1]
```diff
@@ -0,0 +1,169 @@
...
+    triggers:
+    - imageChangeParams:
+        automatic: true
+        containerNames:
+        - syndesis-kudu
+        from:
+          kind: ImageStreamTag
+          name: syndesis-kudu:latest
+      type: ImageChange
+    - type: ConfigChange
```
Comment (avano) [inline utilities/src/main/resources/templates/syndesis-kudu.yml]: add a new line here

**Comment by syndesisqe** [comment]
Build finished. Tests PASSED

**Pull request merged**